src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt

```diff
@@ -111,7 +143,7 @@ class ARCWindow(title: String, task: Task) : JFrame() {
             addActionListener {
                 if (this@ARCWindow.taskType != taskType) {
                     this@ARCWindow.taskType = taskType
-                    refresh(task)
+                    refresh(task, null)
```

Comment:
To avoid forcing the reader to look up the call signature, I'd specify the argument name: 
```
    refresh(task, game = null)
```

Reply:
just used the default